build.ps1

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env pwsh
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+param(
+    [Parameter()]
+    [switch]
+    $Bootstrap,
+
+    [Parameter()]
+    [switch]
+    $Clean,
+
+    [Parameter()]
+    [switch]
+    $Test
+)
+
+$NeededTools = @{
+    VSCode = "Visual Studio Code"
+    NodeJS = "Node.js 6.0 or higher"
+    PowerShellGet = "PowerShellGet latest"
+    InvokeBuild = "InvokeBuild latest"
+}
+
+if ((-not $PSVersionTable["OS"]) -or $PSVersionTable["OS"].Contains("Windows")) {
```

Comment:
Can we just rely on `$IsWindows`?  If it's `$null`, then we can assume it's Windows since that would be returned by Windows PowerShell.  Same for $IsLinux and $IsMacOS.

Reply:
Just realized OS isn't used in this script.